examples/tutorials/nb_python/ECCV_2020_Advanced_Features.py

```diff
@@ -420,43 +421,35 @@ def make_cfg(settings):
             "resolution": [settings["height"], settings["width"]],
```

Comment:
Example of multiple SensorSpec initialization using a dictionary. sut.make_sensor_specs(sensors) and sut.make_sensor_specs_from_settings(sensors, settings) creates the SensorSpecs under the hood

Reply:
"sensor_subtype": habitat_sim.SensorSubType.PINHOLE is added as we need to make sure that the SensorSpec initialized is a CameraSensor